src/components/Header/_header.scss

```diff
@@ -47,3 +47,37 @@
     display: flex;
   }
 }
+
+.#{$prefix}--header__menu-item[role='menuitem'] {
+  display: flex;
+  align-items: center;
+  color: #c6c6c6;
+  height: 100%;
+  font-size: 0.875rem;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: 1.125rem;
+  text-decoration: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  border: 2px solid #0000;
+  transition: background-color 110ms, border-color 110ms, color 110ms;
```

Comment:
Should we use the curves/definitions from [@carbon/motion](https://github.com/carbon-design-system/carbon/tree/master/packages/motion)?

Reply:
yes. probably just copied over. can look into mixins, tokens